src/Subscription.ipp

```diff
@@ -0,0 +1,41 @@
+/**
+ * This software is distributed under the terms of the MIT License.
+ * Copyright (c) 2020 LXRobotics.
+ * Author: Alexander Entinger <alexander.entinger@lxrobotics.com>
+ * Contributors: https://github.com/107-systems/107-Arduino-Cyphal/graphs/contributors.
+ */
+
+/**************************************************************************************
+ * NAMESPACE
+ **************************************************************************************/
+
+namespace impl {
+
+/**************************************************************************************
+ * CTOR/DTOR
+ **************************************************************************************/
+
+template<typename T>
+Subscription<T>::~Subscription()
+{
+  _on_destruction_cb();
+}
+
+/**************************************************************************************
+ * PUBLIC MEMBER FUNCTIONS
+ **************************************************************************************/
+
+template<typename T>
+void Subscription<T>::onTransferReceived(CanardRxTransfer const & transfer)
+{
+  T const msg = T::deserialize(transfer);
```

Comment:
Error handling seems to be missing.

Reply:
Guilty, but that's really a problem of the current C++ wrapper system, see [here](https://github.com/107-systems/107-Arduino-Cyphal/issues/185#issuecomment-1363968936). Will be resolved in separate PR.